src/EditorFeatures/Core/Implementation/InlineRename/Dashboard/Dashboard.xaml.cs

```diff
@@ -62,11 +63,27 @@ internal partial class Dashboard : UserControl, IDisposable
                 // Find UI doesn't exist in ETA.
             }
 
+            this.Loaded += Dashboard_Loaded;
```

Comment:
Can you add a comment why this has to be done in an event vs. just the constructor?